### DISCUSSION TITLE
sspai

### DISCUSSION THREAD

**Discussion by queyuewutong** [gildas-lormeau/SingleFile#1149] [Q&A]
When I used the SingleFile plugin in CentBrowser to clip articles from the website "[少数派](https://sspai.com/)" in the past, the layout was messed up after clipping. The Chromium kernel version of CentBrowser was 102, and the SingleFile version was 1.21.39. My computer was running on the Windows 10 operating system.

**Comment by gildas-lormeau** [comment]
Please try to enable `Misc > pass "Referer" header after a cross-origin request error` in the options page of SingleFile.

**Comment by queyuewutong** [comment]
I‘m sorry,but I can’t find Misc > pass "Referer" header after a cross-origin request error in my singlefile.It does't exixt.

**Comment by gildas-lormeau** [comment]
My bad,  the option `Misc > pass "Referer" header after a cross-origin request error` is in the `Network` panel.

**Comment by queyuewutong** [comment]
Thank you for your response. The error on my end has been fixed.

**Comment by queyuewutong** [comment]
Thank you for your response. The error on my end has been fixed.

**Comment by gildas-lormeau** [comment]
I'm glad to hear it :)
